tests/api/test_ratelimiting.py

```diff
@@ -5,35 +5,25 @@
 
 class TestRatelimiter(unittest.TestCase):
     def test_allowed(self):
```

Comment:
We should add tests for the situation when the `rate_hz` / `burst_count` are overridden.

Reply:
Tests galore!